### ISSUE TITLE
Stale issues automation / bot

### ISSUE THREAD

**Issue by nuclearcat** [kernelci/kernelci-project#467]
We have A LOT of issues, many of which are stale. I think we need automatically remind if there is no motion on issue, that updates required, with option to postpone reminder up to 6 month.
Any opinion?
@padovan @broonie @pawiecz @JenySadadia

**Comment by nuclearcat** [comment]
Probably pull requests as well

**Comment by JenySadadia** [comment]
Yes, sounds like a good idea to me.
Does GH has that kind of feature?

**Comment by nuclearcat** [comment]
Yes, i saw such on some projects, will try to figure out how they did it.
